### PR TITLE
AG-8052 Simplify Selection

### PR DESCRIPTION
**Pull request by alexanderby**
https://ag-grid.atlassian.net/browse/AG-8052

### CODE REVIEW

File: charts-community-modules/ag-charts-community/src/axis.ts [hunk 1]
```diff
@@ -715,12 +716,12 @@ export class Axis<S extends Scale<D, number, TickInterval<S>>, D = any> {
             if (visible) {
                 anyTickVisible = true;
             }
-            return visible;
+            node.visible = visible;
```
Comment (alantreadway) [inline charts-community-modules/ag-charts-community/src/axis.ts:719]: Looks slightly suspicious since removing `return` alters the control flow?

**Review by alantreadway — COMMENTED** [review]
Generally LGTM - I'll check out the branch and kick the wheels before merging a bit later today if everything checks out (I'd rather get this merged earlier in the release cycle to give us some chance of finding unforeseen side-effects before release).

File: charts-community-modules/ag-charts-community/src/axis.ts (hunk 1 above)
Comment (alexanderby) [reply charts-community-modules/ag-charts-community/src/axis.ts:719]: It is the end of the function, so the return is unnecessary.

Comment (alexanderby) [reply charts-community-modules/ag-charts-community/src/axis.ts:719]: So, previously this was a callback for the `attr()` method, but now it is a callback for the `each()` method.

**Comment by alantreadway** [comment]
I made some fixes to Sparklines (it was also using `Selection` heavily), but this is working without too much effort as it was following the exact same patterns as Standalone Charts.

**Pull request merged**